recipes/carnac-lr/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "CARNAC-LR" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "13601b79b01f83348a725edf5414d8b4916d21428f98ed335195ca6d446f7e8a" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/kamimrcht/CARNAC-LR/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - makefile.patch
+    - python_path.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - python >=3
+
+test:
+  files:
```

Comment:
is this file now needed?

Reply:
The test file or the makefile patch?

Reply:
the test file...

Reply:
The test file is needed for the test in run_test.sh ("CARNAC-LR -f test.input -o output.txt
"). Should I not include test at all? The file is 3.2KB, so I thought having the test is a good practice?